src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -169,43 +174,47 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
       }
     ];
 
+    const isCurrentPermTypeDelegated = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;
```

Comment:
I feel like the naming of this variable is a bit weird, since `tokenPresent` serves a similar purpose of describing the state but is worded differently. Maybe `isCurrentPermTypeDelegated` -> `delegatedPermissions` serves to more concisely describe the state in the same manner as `tokenPresent` does?

Reply:
yeah I agree!

Reply:
I feel it's weird to compare two Booleans and assign it to anther Boolean. Had a comment above regarding making `permissionModeType` as an enum. Then `DISPLAY_DELEGATED_PERMISSIONS` and `DISPLAY_APPLICATION_PERMISSIONS` should be able to be replaced by the enum? (Correct me if I'm wrong lol)

Reply:
@jerryxihe You could even make it `isDelegatedPermissions` actually if you want extra clarity; It doesn’t necessarily have to match the older codebase items since refactoring stuff is pretty normal

Reply:
c99a5c534eba1492284a1e3e4a9f34c2193d5eb8